Doc/library/importlib.rst

```diff
@@ -1240,6 +1240,16 @@ an :term:`importer`.
    .. versionchanged:: 3.6
       Accepts a :term:`path-like object`.
 
+.. function:: load_source_path(module_name, filename)
+
+   Load a module from a filename: execute the module and add it to
+   :data:`sys.modules`.
+
+   A package cannot be imported by its directory path, whereas its
+   ``__init__.py`` file (ex: ``package/__init__.py``) can be imported.
+
+   .. versionadded:: 3.12
```

Comment:
The PR has no `needs backport to 3.12` label. Should the version be 3.13 instead?

```suggestion
   .. versionadded:: 3.13
```

Reply:
Also, it seems there is no support for the backport to satisfy a specific use case:

> As I said over on discuss.python.org, Miro's ask is actually different from yours. You want to load from a specific path, Miro wants to search and load from a specific directory (note how your use case lacks a search component).